juju/juju-gui-charm
# Fix charm not deploying correctly on xenial.

This was due to a pip install error.
Also:
- print the password when "make deploy" completes;
- add additional clean-up of unused helper functions;
- fix "make help" command descriptions.

QA:
Ensure the GUI charm can be correctly deployed:
run both "make deploy" and "make deploy SERIES=trusty".
Also check that the password is correctly printed when
the unit is ready.
